opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -499,7 +500,7 @@ def _format_links(links):
         return f_links
 
 
-class _Limits:
+class SpanLimits:
```

Comment:
<img width="794" alt="Screenshot 2021-05-28 at 8 36 57 AM" src="https://user-images.githubusercontent.com/22846633/119923773-f4030800-bf8f-11eb-8282-11e50f4b4f4b.png">
This description doesn't seem to be accurate. It says setting limit to `None` will not set any limits but the implementation sets the default length if value is None.

Reply:
Thanks. Too many iterations :)